ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.15__amd64-stable.yaml

```diff
@@ -128,8 +128,8 @@ tests:
     test:
     - chain: cucushift-installer-check-cluster-health
     workflow: cucushift-installer-rehearse-aws-ipi-disconnected-private-cco-manual-security-token-service-private-s3-with-ep-sts-ec2-elb
-- as: aws-ipi-ovn-ipsec-to-multiarch-f14
-  cron: 55 7 4,18 * *
+- as: aws-ipi-ovn-ipsec-to-multiarch-f28
```

Comment:
Shall we enable OCP-60397 on the old versions ?

Reply:
oh :facepalm: I missed .15 .16 on x86. since already started the rehearse, I'll wait for it to finish and then fix